### DISCUSSION TITLE
posts/2024/a83bb32b/

### DISCUSSION THREAD

**Discussion by giscus** [GZTimeWalker/blog-comments#25] [Announcements]
# posts/2024/a83bb32b/

风尘仆仆又满是回忆的一年

https://blog.gzti.me/posts/2024/a83bb32b/

**Comment by GrakePch** [comment]
祝好运🚀

**Comment by NorthSecond** [comment]
生日快乐！🍻

感觉这种状态确实让您状态好了不少。祝好运！

<del>欢迎没事溜达去汉中玩，回不去我也能线上给您说一下有啥好吃的</del>

**Comment by yemq3** [comment]
生日快乐🥳

**Comment by shansing** [comment]
生日快乐

**Comment by Ray-Eldath** [comment]
生日快乐~ 祝前途光明！

**Comment by tomatoyuan** [comment]
祝平安喜乐，万事顺遂😊

**Comment by Steaming-Fresh** [comment]
23，质数，象征独特、稳定、幸福
也一定能新岁新生，万事如意
祝生日快乐🎂

**Comment by zhangrayhung** [comment]
现在的就业机会都集中在了北上广深，真的发展太畸形，你很难选择一个自己喜欢的城市去工作

**Comment by RunningKuma** [comment]
半夜睡不着，把学长的博客刷了一遍
谢谢你

**Comment by hxy860721** [comment]
你好 我是陕西汉中这边的  请联系下我   我有CTF比赛相关事宜寻求合作   40323484@qq.com
